src/04-modality-specific-files/01-magnetic-resonance-imaging-data.md

```diff
@@ -447,6 +447,135 @@ JSON example:
+| BackgroundSuppression             | REQUIRED                                                                                                                        | [boolean][]                             | Boolean indicating if background suppression is used.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                               |
+| M0                                | REQUIRED                                                                                                                        | [number][] or [string][] or [boolean][] | Describes the available M0 information or the location of an `m0scan` file, as either: *Numerical*: a single numerical whole-brain M0 value, for example obtained from CSF in another sequence; *SeparateFile*: `*_m0scan.nii[.gz]`; *Within ASL time-series*: *Boolean: `true`* referring to an `m0scan` within the time-series (as defined within the `*_aslcontext.tsv`); *`false`* referring to the use of the (average) `control` image for M0 calibration.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                    |
```

Comment:
```suggestion
| M0                                | REQUIRED                                                                                                                        | [number][] or [string][] or [boolean][] | Describes the available M0 information or the location of an `m0scan` file, as either: 
- *Numerical*: a single numerical whole-brain M0 value, for example obtained from CSF in another sequence;
- *SeparateFile*: `*_m0scan.nii[.gz]`; 
- *Within ASL time-series*: 
- *Boolean: `true`* referring to an `m0scan` within the time-series (as defined within the `*_aslcontext.tsv`);
- *Boolean: `false`* referring to the use of the (average) `control` image for M0 calibration.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                    |
```

Reply:
From separate discussion of `M0.nii.gz`, this might change from `M0`. What I've suggested above would be `M0Estimate`. You could also add an explicit `M0Type` that could be a string such as "Scan", "Volume", "Estimate", corresponding to current "SeparateFile`, `true`, `<number>` to indicate which place to look, if the implicit logic is uncomfortable.

Overall point is that I think two variables are better than one highly overloaded one.

Reply:
@effigies  Thanks for the feedback and suggestions. Indeed, I tried the bullet points, but that doesn't work with table. 
I understand that the current options for the M0 might be a bit much. However, I feel that the strategy 'M0 optional' you proposed might lead to datasets for which M0 is not defined correctly. Additionally, a seperately acquired scan should always be preferred. However, splitting up the fields might be a better option. I will definitely put this on the agenda of the meeting this week. Let's see if we can find another solution for this, based on your suggestions.

Reply:
I like the solution `M0Type`: "Scan", "Volume", "Estimate" (we don't need to specify the "fallback to average control image"). And then we have indeed a separate optional numeric field `M0Estimate` that is required when M0Type equals 'Estimate'. @patsycle let's indeed discuss this internally